docs/source/using/experimental-features.adoc

```diff
@@ -87,3 +87,29 @@ PS> minishift.exe start `
 Be sure to specify a valid gateway and nameserver.
 Failing to do so will result in connectivity issues.
 ====
```

Comment:
@gbraad , I think we also need to update `Assign IP Address to Hyper-V` section https://github.com/gbraad/minishift/blob/15f9793a6462c56e0795bf1ad43b883e7425d755/docs/source/using/experimental-features.adoc#assign-ip-address-to-hyper-v.

There is a line `To make this work you need to create a Virtual Switch using NAT.` which need to update since user can have `Default Switch`.

Reply:
i'll add 'or us the Default Switch'. But will not explain more at this stage. Default Switch is still not perfectly working.

Reply:
I'd rather not write this on review. Support for the Default Switch is not optimal. We can make it work to get an IP address, but static assigned addresses do not work on my setup, but it did work for @anjannath. I'd rather check this fully after fixing #2096 and add the right documentation.